### DISCUSSION TITLE
Full automatic login

### DISCUSSION THREAD

**Discussion by Sfeeen** [ahivert/tgtg-python#203] [Ideas]
Is there also some code to provido a full login. 
So maybe a function to which you pass the confirm url you receive by mail -> https://share.toogoodtogo.com/login/accept/{user_id}/{token}
And then does the necessary to retrieve a bearer?
Would be awesome!
Tried it myself but got into captcha's... assumingly because of the datadome cookie...

**Comment by cazicruz** [comment]
i believe there is ..... working on something similar but haven't achieved this yet hopefully soon
